vector/v.external.out/v.external.out.html

```diff
@@ -67,7 +67,7 @@ <h3>PostgreSQL/PostGIS Creation Options</h3>
   default: <tt>topo</tt></li>
   <li><tt>TOPO_TOLERANCE=&lt;value&gt;</tt> - tolerance for PostGIS
   Topology schema,
-  see <a href="http://www.postgis.net/docs/manual-2.0/CreateTopology.html">CreateTopology</a>
+  see <a href="https://postgis.net/docs/manual-2.0/CreateTopology.html">CreateTopology</a>
```

Comment:
Versionless: https://postgis.net/docs/CreateTopology.html
For version 3.5: https://postgis.net/docs/manual-3.5/CreateTopology.html

Reply:
Thanks, replaced with versionless URL.